client/rest-high-level/src/test/java/org/elasticsearch/client/SearchIT.java

```diff
@@ -1227,7 +1227,7 @@ public void testFieldCaps() throws IOException {
         FieldCapabilitiesResponse response = execute(request,
             highLevelClient()::fieldCaps, highLevelClient()::fieldCapsAsync);
 
-        assertEquals(new String[] {"index1", "index2"}, response.getIndices());
+        assertArrayEquals(new String[] {"index1", "index2"}, response.getIndices());
```

Comment:
Since we are changing this, I think using hamcrest will produce a nicer error message if this ever fails:
```
assertThat(response.getIndices(), Matchers.arrayContaining("index1", "index2"));
```

Reply:
Good point — I'll switch it.